sonar/models/mutox/classifier.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the license found in the
+# MIT_LICENSE file in the root directory of this source tree.
+
+from dataclasses import dataclass
+import torch
+from torch import nn
+
+from fairseq2.typing import DataType, Device
+from fairseq2.models.utils.arch_registry import ArchitectureRegistry
+
+from typing import Optional
+
+
+class MutoxClassifier(nn.Module):
+    def __init__(
+        self,
+        model_all,
+    ):
+        super().__init__()
+        self.model_all = model_all
+
+    def forward(self, inputs: torch.Tensor) -> torch.Tensor:
+        return self.model_all(inputs)
```

Comment:
forward seems to be trivial here, it looks like model_all =  MutoxClassifier...
maybe we could move model layer creation from the builder to MutoxClassifier to make it more meaningful.